packages/reputation-miner/ReputationMiner.js

```diff
@@ -12,6 +12,7 @@ const sqlite = require("sqlite");
 // const accountAddress = "0xbb46703786c2049d4d6dd43f5b4edf52a20fefe4";
 const secretKey = "0xe5c050bb6bfdd9c29397b8fe6ed59ad2f7df83d6fd213b473f84b489205d9fc7";
 const miningCycleDuration = 60 * 60 * 24; // 24 hours
+const minStake = ethers.utils.bigNumberify(10).pow(18).mul(2000); // eslint-disable-line prettier/prettier
```

Comment:
How would you feel about including a `constants` for the reputation miner? Or including them here? Having `WAD` and `UINT256_MAX` constants would make some of this code clearer.